src/componets/Header/styles.jsx

```diff
@@ -0,0 +1,85 @@
+import styled, { keyframes } from 'styled-components';
+import { ReactComponent as PokeballIcon } from '../../assets/pokeball.svg';
+
+export const HeaderContainer = styled.div`
+    width: 100%;
+    height: 100px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    flex-direction: column;
+`;
+export const HeaderBar = styled.div`
+    width: 76%;
+    height: 70px;
+    background: white;
+    border-radius: 10px;
+    box-shadow: 10px 10px 10px rgba(0, 0, 0, 0.3);
+    display: flex;
+    flex-direction: row;
+    justify-content: left;
+    align-items: center;
+    z-index: 0;
+    border: 2px solid #EEEE;
+    min-width: 350px;
+`;
+export const IconPokeball = styled(PokeballIcon)`
+    width: 25px;
+    height: 25px;
+    margin-right: 10px;
+`;
+
+export const Line = styled.span`
+    height: 3px;
+    position: absolute;
+    bottom: 0;
+`;
+const growBar = keyframes`
+    from{
+        width: 0;
+    }
+    to{
+        width: 100%;
+    }
+`;
+export const HeaderSection = styled.div`
+    width: 110px;
+    height: 70px;
+    display: flex;
+    position: relative;
+    flex-direction: row;
+    color: #80808080;
+    justify-content: center;
+    align-items: center;
+    cursor: pointer;
+    font-family: 'Rubik', sans-serif;
+    font-weight: 900;
+    border-radius: 5px;
+    margin: 0 10px;
+    &:hover{
```

Comment:
use a undefined width, and instead of margin use paddings to not see the space between bars on the menu